### DISCUSSION TITLE
RUnning bot in replit.com

### DISCUSSION THREAD

**Discussion by SparrowWatt** [sleeyax/aternos-discord-bot#75] [Q&A]
Hello, 
Fnice job with the bot but is there any way to host it with **replit**? If yes, could you make a tutorial about that. Thank you very much!

**Comment by sleeyax** [comment]
It *might* work but (apparently) only if you have a paid account on replit. Check out replit docs about how to run go applications.

**Reply by SparrowWatt** [reply]
Thank you, also, is there any chance that the bot will not be selfhost? I mean that someone will run it for all, or is it just impossible? Thank you very much for your time

**Reply by sleeyax** [reply]
I initially hosted a public bot but people kept whining about their accounts being banned once it gained traction and atenos kept banning my VPS IPs. It could technically still be possible to host a public instance by serving each connection through a residential HTTP proxy though, but that would get very expensive quickly.

See also: https://github.com/sleeyax/aternos-discord-bot/discussions/61
And: https://github.com/sleeyax/aternos-api/issues/4
